Commands.adoc

```diff
@@ -0,0 +1,239 @@
+
+To check the routable IP address of the VM:
+---------------------------------
+$ vagrant service-manager box ip
+172.28.128.3
+---------------------------------
+
+To check the routable IP address of the VM in key=value format:
+---------------------------------------------------
+$ vagrant service-manager box ip --script-readable
+IP=172.28.128.3
+---------------------------------------------------
+
+To check the version of the VM:
+--------------------------------------
+$ vagrant service-manager box version
+Atomic Developer Bundle (ADB) 2.2.0
+--------------------------------------
+
+To check the version of the VM in key=value format:
+--------------------------------------------------------
+$ vagrant service-manager box version --script-readable
+VARIANT="Atomic Developer Bundle (ADB)"
+VARIANT_ID="adb"
+VARIANT_VERSION="2.2.0"
+--------------------------------------------------------
+
+
+== Manage Service Lifecycles
+
+If you need to mange the lifecycle of a service - check the current status of
+existing services, start or stop or restart a specific service use: +
+`$ vagrant service-manager [sub-command] [service]`
+
+The following sub-commands can be used: +
+`status`:   Lists services and their state (running/stopped) +
+`start`:   Starts the given service in the VM +
+`stop`:   Stops the given service in the VM +
+`restart`:   Restarts the given service in the VM +
+
+The possible options for service are `docker`, `openshift` and `kubernetes`.
+
+=== Examples
+
+If no service is specified `vagrant service-manager status` will provide
+information on all the configured services and their state (running/stopped).
+---------------------------------------
+$ vagrant service-manager status
+Configured services:
+docker - stopped
+openshift - running
+kubernetes - stopped
+---------------------------------------
+
+To check the status of Docker:
+-------------------------------------------
+$ vagrant service-manager status docker
+docker - stopped
+-------------------------------------------
+
+To start Docker:
+---------------------------------------------
+$ vagrant service-manager start docker
+$ vagrant service-manager status docker
+docker - running
+---------------------------------------------
+
+To stop Docker:
+---------------------------------------------
+$ vagrant service-manager stop docker
+$ vagrant service-manager status docker
+docker - stopped
+---------------------------------------------
+
+To restart Docker:
+-----------------------------------------
+$ vagrant service-manager restart docker
+$ vagrant service-manager status docker
+docker - running
+-----------------------------------------
+
+Similarly, you can use `$ vagrant service-manager status openshift`,
+ `$ vagrant service-manager start openshift`, `$ vagrant service-manager stop openshift`,
+ and `$ vagrant service-manager restart openshift`, to verify the status, start,
+  stop and restart OpenShift.
+
+== Use the vagrant-service-manager Help
```

Comment:
Changed the section title to Use the Help. However, unable to add empty line after heading, even after using ' +' and '[%hardbreak]'. Probably something to do with asciidoc's heading style. On rendering, I did not feel the need for an additional line as it does leave a fair amount of space between heading and text.
In Readme, I put the help command first as it is a gist of the commands, and a seasoned user may simply want to glance and take a stab himself. But for this doc, I feel the user would have seen the help command in the readme and still be interested in the details of the available commands, and hence visiting it. So I feel explaining all the available commands first and then the help command, makes sense here.